### PR TITLE
BCF-2147: fix admin login command

### PR DESCRIPTION
**Pull request by krehermann**
Was missing a MkdirAll. Add a test.

### CODE REVIEW

File: core/cmd/client_test.go [hunk 1]
```diff
@@ -305,3 +310,17 @@ func TestFileSessionRequestBuilder(t *testing.T) {
 		})
 	}
 }
+
+func TestNewUserCache(t *testing.T) {
+
+	r, err := rand.Int(rand.Reader, big.NewInt(256*1024*1024))
```
Comment (jmank88) [inline core/cmd/client_test.go:316]: Unchecked err?

File: core/cmd/client_test.go [hunk 2]
```diff
@@ -305,3 +310,17 @@ func TestFileSessionRequestBuilder(t *testing.T) {
...
+	subDir := filepath.Base(fmt.Sprintf("%s-%d", t.Name(), r.Int64()))
```
Comment (jmank88) [inline core/cmd/client_test.go:317]: Would it make sense to use `t.TempDir()`? https://pkg.go.dev/testing#T.TempDir

Comment (krehermann) [reply core/cmd/client_test.go:321]: no, t.TempDir and ioutil.TempDir don't fit well here. Added a comment.

File: core/cmd/client_test.go (hunk 1 above)
Comment (krehermann) [reply core/cmd/client_test.go:316]: thanks. done

Comment (jmank88) [reply core/cmd/client_test.go:316]: Did the linter flag this BTW? We might be omitting _test files or something - should reconsider that.

Comment (krehermann) [reply core/cmd/client_test.go:316]: it did not. i agree.

**Pull request merged**